examples/aasp/README.md

```diff
@@ -0,0 +1,182 @@
+# Microsoft Azure Attestion Secret Provisioning(AASP)
+
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Secret Provisioning Workflow](#secret-provisioning-workflow)
+    - [Generate a private/public key pair and prepare for the key release policy](#generate-a-key-pair-and-prepare-for-the-key-release-policy)
+    - [Generate a wrapped secret using the public key using the aasp binary](#generate-a-wrapped-secret-using-the-public-key-using-the-aasp-binary)
+    - [Running the AASP container in a TEE](#running-the-aasp-container-in-a-tee)
+    - [Unwrap the secret using grpcurl](#unwrap-the-secret-using-grpcurl)
+
+
+# Introduction:
```

Comment:
Can you update the format of this README to be more step-by-step with step 1, step 2, and etc

Reply:
this is done